Movement.cpp

```diff
@@ -9,105 +9,105 @@
 Movement::Movement() {}
 
 void Movement::advance(const double desire, const double desireUltrasonic) {
-  double encoder1 = 0;
-  double encoder2 = 0;
-  double pwm_left_final = 0;
-  double pwm_right_final = 0;
-  double pwm_left = 0;
-  double pwm_right = 0;
-  double pwm_right_BNO = 0;
-  double pwm_left_BNO = 0;
-  double pwm_right_ultrasonic_right_up = 0;
-  double pwm_left_ultrasonic_right_up = 0;
-  double pwm_left_ultrasonic_right_down = 0;
-  double pwm_right_ultrasonic_right_down = 0;
-  double pwm_left_ultrasonic_left_up = 0;
-  double pwm_right_ultrasonic_left_up = 0;
-  double pwm_left_ultrasonic_left_down = 0;
-  double pwm_right_ultrasonic_left_down = 0;
-  const double current_distance_right_up = sensor.getDistanceRightUp();
-  const double current_distance_right_down = sensor.getDistanceRightDown();
-  const double current_distance_left_up = sensor.getDistanceLeftUp();
-  const double current_distance_left_down = sensor.getDistanceLeftDown();
-  double error_right_up = control.getErrorUltrasonic(current_distance_right_up, desireUltrasonic);
-  double error_right_down = control.getErrorUltrasonic(current_distance_right_down, desireUltrasonic);
-  double error_left_up = control.getErrorUltrasonic(current_distance_left_up, desireUltrasonic);
-  double error_left_down = control.getErrorUltrasonic(current_distance_left_down, desireUltrasonic);
-  double current_angle = control.getAngleBNOX();
-  double errorBNO = control.getAngleError(current_angle, desire);
+  eCount1 = 0;
+  eCount2 = 0;
```

Comment:
encoder_count_left, encoder_count_right, or however you want to call them. Simply follow the first_second_hi_bla format.